imgs/baseRounded.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Could we not call this file something like 'logoSmall'? The current name gives me no indication as to what this image actually is.